### PR TITLE
#7 standardize stripe endpoint

### LINKED ISSUE #7: Standardize Stripe Endpoint
This task is set to standardize the Stripe Endpoint.

Remember to add two or three important business flow steps.

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull request for the topic https://github.com/slingr-stack/stripe-endpoint/issues/7 created by @agreggio-slingr

## What does it do?
This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and step flows.

## How to test it?
Make calls from the platform for example using the console using the information provided in the documentation.

## Implementation notes
Create new version of the endpoint, it is not compatible with previous versions.

## Implementation notes
Create a new tags changing the major version and deploy to the platform.

### CODE REVIEW

File: gen/helperGen.json [hunk 1]
```diff
@@ -0,0 +1,2262 @@
...
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints/:webhook_endpoint"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints/:webhook_endpoint"
```
Comment (pasaperez-slingr) [inline gen/helperGen.json:2258]: Here in the gen can be added:

`"omittableUrlParts": [
        "v1"
      ],`
      
This way the helpers are simplified a little bit.

Comment (pasaperez-slingr) [reply gen/helperGen.json:2258]: I'm not sure now how it works, but as an alternative, in order not to modify each path, you can try "omittableUrlParts" at the 1st level, as the "endpointName".

**Comment by pasaperez-slingr** [comment]
#approve

**Comment by sebasmoyano** [comment]
#approve

**Pull request merged**